### PR TITLE
Turning the 'Contact Sales' button into a mailto link

### LINKED ISSUE #727: Make 'Contact Sales' option visible before end of trial
Currently in the license/registration dialog the Contact Sales option is disabled until the extended trial has expired. This option needs to remain visible throughout in case user wants to contact Sales even before the end of trial.

Connects to https://github.com/Particular/CustomerSuccess/issues/524

### PR DESCRIPTION
**Pull request by HEskandari**
Closes #727

### CODE REVIEW

File: src/ServiceInsight/Shell/LicenseRegistrationViewModel.cs [hunk 1]
```diff
@@ -123,7 +121,7 @@ public void Extend()
 
         public void ContactSales()
         {
-            network.Browse("http://particular.net/extend-your-trial-45");
+            network.Browse("mailto:contact@particular.net?subject=ServiceInsight");
```
Comment (WilliamBZA) [inline src/ServiceInsight/Shell/LicenseRegistrationViewModel.cs]: What happens if no mail client is installed?

Comment (WilliamBZA) [reply src/ServiceInsight/Shell/LicenseRegistrationViewModel.cs]: https://stackoverflow.com/questions/27176849/what-does-mailto-do-when-there-is-no-email-client

TL;DR: I don't think it's something we should concern ourselves with.

Comment (HEskandari) [reply src/ServiceInsight/Shell/LicenseRegistrationViewModel.cs]: When balancing effort vs. value we decided to go with mailto instead of a webform on the website, so this is an acceptable risk (although Windows and Mac both come with mail clients preinstalled)
